File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        d ={}
        for i in nums:
            if i in d:
                d[i] +=1
            else:
                d[i] = 1
        for i in d:
            if d[i]>1:
                for j in range(d[i]-1):
                    nums.remove(i)
        return len(nums)

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.kt
class Solution {
    fun removeDuplicates(nums: IntArray): Int {
        var r = 0; var l = 0; var i = 0;
        while(r<nums.size){
            while(r<nums.size && nums[r] == nums[l]){
                r++;
            }
            nums[i] = nums[l]
            i++
            l = r
        }
        return i
    }
}